### PR TITLE
Jericho proposed code

### PR DESCRIPTION
**Pull request by tmf-gsmith**

### CODE REVIEW

File: Code/STAGE1bih_merging_and_organization.R [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#VERSION = "C"
+#VERSION = "multi_2017"
+
+# Array of subjects.
+if (VERSION == "M"){
+  subjects = c('04015', '04043', '04048', '04126', '04746', '04908', '04936', '05091', '05121', 
+               '05261', '06426', '06453', '06995', '07162', '07859', '07879', '07910', '08215', 
+               '08219', '08378', '08405', '08434', '08455')
+}else{
+  subjects = read.csv("C:/Users/mario/Documents/Internship and REU Stuff/UNCW 2019/Atrial_Fibrillation/Testing/training2017/REFERENCE_2.csv", header = F)$V2[-1]
```
Comment (tmf-gsmith) [inline Code/STAGE1bih_merging_and_organization.R]: We need to have a variable with the file path of data that the user can change. Then we just append the file name to it where we call it.

File: Code/STAGE1bih_merging_and_organization.R [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+## Merging and Organization
+
+# Adds calculated features to the data gathered from the MIT-BIH/2017 subjects.
+# Merges all data into one file.
+
+## Pick the correct version for testing. ##
+VERSION = "M"
+#VERSION = "C"
+#VERSION = "multi_2017"
+
```
Comment (tmf-gsmith) [inline Code/STAGE1bih_merging_and_organization.R]: file_path = "C:/Users/mario/Documents/Internship and REU Stuff/UNCW 2019/Atrial_Fibrillation/Testing"

File: Code/STAGE1get_that_wfdb.m [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+%% Summer 2019, 2021
+%% Extracting AFib Data
+
+% Using the mcode from the training2017 folder of the training2017.zip 
+% file, the code extracts annotations from the 2017 PhysioNet/CinC 
+% challenge and generates basic RR-interval information for each of 
+% the subjects.
+
+% Directory where all files are located
+SOURCE_FROM = "/Users/mario/Documents/Internship and REU Stuff/UNCW 2019/Atrial_Fibrillation/Testing/training2017";
```
Comment (tmf-gsmith) [inline Code/STAGE1get_that_wfdb.m]: We need to SOURCE_FROM in every doc

Comment (jericholawson) [reply Code/STAGE1get_that_wfdb.m]: I'll double-check that I added those suggestions in there. I updated the first two stages of code to make the process of changing file names/globals smoother. I'm still in the process of refining the code, but the functionality should remain the same in the updated code I just committed.

**Pull request merged**